proto/buf.gen.swagger.yaml

```diff
@@ -4,5 +4,5 @@ version: v1
 
 plugins:
   - name: swagger
-    out: ../tmp-swagger-gen
+    out: ../app/client/docs
```

Comment:
This path is valid on regen but not on our network. Maybe publish this on a top level folder called `api`